docs/dev/modules.md

```diff
@@ -49,14 +50,16 @@ This page is generated via script `./gradlew :tools:dev:renderModuleDetails`. Se
 
 | Module | Description | Module Status |
 |---	|---	|---    |
-| [incubator-openwhisk-runtime-ballerina](https://github.com/apache/incubator-openwhisk-runtime-ballerina) | Apache OpenWhisk Ballerina runtime| [![Build Status](https://travis-ci.org/apache/incubator-openwhisk-runtime-ballerina.svg?branch=master)](https://travis-ci.org/apache/incubator-openwhisk-runtime-ballerina) |
-| [incubator-openwhisk-runtime-docker](https://github.com/apache/incubator-openwhisk-runtime-docker) | Apache openwhisk | [![Build Status](https://travis-ci.org/apache/incubator-openwhisk-runtime-docker.svg?branch=master)](https://travis-ci.org/apache/incubator-openwhisk-runtime-docker) |
+| [incubator-openwhisk-runtime-ballerina](https://github.com/apache/incubator-openwhisk-runtime-ballerina) | Apache OpenWhisk Ballerina runtime. Learn more at https://openwhisk.apache.org.  | [![Build Status](https://travis-ci.org/apache/incubator-openwhisk-runtime-ballerina.svg?branch=master)](https://travis-ci.org/apache/incubator-openwhisk-runtime-ballerina) |
+| [incubator-openwhisk-runtime-docker](https://github.com/apache/incubator-openwhisk-runtime-docker) | Apache OpenWhisk SDK for building Docker “blackbox" runtimes. https://openwhisk.apache.org/ | [![Build Status](https://travis-ci.org/apache/incubator-openwhisk-runtime-docker.svg?branch=master)](https://travis-ci.org/apache/incubator-openwhisk-runtime-docker) |
```

Comment:
Wondering why the link to https://openwhisk.apache.org/ in only some of the table entries?

Reply:
This is picked from the description field. Some of them have link there and some do not

```json
 {
        "id": 105323132,
        "node_id": "MDEwOlJlcG9zaXRvcnkxMDUzMjMxMzI=",
        "name": "incubator-openwhisk-runtime-nodejs",
        "full_name": "apache/incubator-openwhisk-runtime-nodejs",
        "private": false,
        "owner": {
            "login": "apache",
            "id": 47359,
            "node_id": "MDEyOk9yZ2FuaXphdGlvbjQ3MzU5",
            "avatar_url": "https://avatars0.githubusercontent.com/u/47359?v=4",
            "gravatar_id": "",
            "url": "https://api.github.com/users/apache",
            "html_url": "https://github.com/apache",
            "followers_url": "https://api.github.com/users/apache/followers",
            "following_url": "https://api.github.com/users/apache/following{/other_user}",
            "gists_url": "https://api.github.com/users/apache/gists{/gist_id}",
            "starred_url": "https://api.github.com/users/apache/starred{/owner}{/repo}",
            "subscriptions_url": "https://api.github.com/users/apache/subscriptions",
            "organizations_url": "https://api.github.com/users/apache/orgs",
            "repos_url": "https://api.github.com/users/apache/repos",
            "events_url": "https://api.github.com/users/apache/events{/privacy}",
            "received_events_url": "https://api.github.com/users/apache/received_events",
            "type": "Organization",
            "site_admin": false
        },
        "html_url": "https://github.com/apache/incubator-openwhisk-runtime-nodejs",
        "description": "Apache OpenWhisk runtime for Node.js JavaScript language functions. https://openwhisk.apache.org/",
```